Makefile

```diff
@@ -99,7 +99,7 @@ check:
 
 test:
 	go test -short -tags='debug testing netgo' -timeout=5s $(pkgs) -run=. -count=$(count)
-test-long: clean fmt vet lint-ci
+test-long: clean fmt vet lint-ci test
```

Comment:
You shouldn't need to include `test` here. That is just going to run the short tests twice.
Long tests are skipped unless it is a long test run, but short test shouldn't have any skip code in them

Reply:
Oh, you're right. :)